virttest/env_process.py

```diff
@@ -677,14 +678,37 @@ def preprocess(test, params, env):
     if params.get("storage_type") == "nfs":
         image_nfs = nfs.Nfs(params)
         image_nfs.setup()
+        if params.get("migration_setup", "no") == "yes":
```

Comment:
if params.get("migration_setup", "no") == "yes":
The statement was called more than twice,suggest use boolean variable to initialize it at the beginning.